benchmark/first_paint/chartjs-master.html

```diff
@@ -14,8 +14,8 @@
 <body>
     <canvas id="chart" width="960" height="720"></canvas>
 
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.min.js"></script>
-    <script src="https://www.chartjs.org/dist/master/Chart.js"></script>
+    <script src="https://www.chartjs.org/dist/master/chart.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chartjs-adapter-date-fns@1.1.0-alpha.3/dist/chartjs-adapter-date-fns.bundle.min.js"></script>
```

Comment:
Why use `date-fns` instead of `moment.js`? From the [official sample](https://github.com/chartjs/Chart.js/blob/master/samples/scales/time/line.html), it seems `moment.js` is preferred. And I don't think this will affect the performance much.

```suggestion
    <script src="https://www.chartjs.org/dist/master/chart.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.28.0/moment.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/chartjs-adapter-moment/0.1.2/chartjs-adapter-moment.min.js"></script>
```

Reply:
Moment is there for historical reasons, I doubt its preferred. But it makes very little difference when `parsing` is set to false, so reverted.